website/blog/2022-03-14-an-update-on-the-new-architecture-rollout.md

```diff
@@ -0,0 +1,114 @@
+---
+title: An update on the New Architecture Rollout
+authors: [cortinico]
+tags: [announcement]
```

Comment:
```suggestion
tags: [announcement]
date: 2022-03-15
```

Probably the date should be adjusted before publishing:
https://docusaurus.io/docs/next/api/plugins/@docusaurus/plugin-content-blog#date